docs/feature_rgb_matrix.md

```diff
@@ -150,6 +150,13 @@ These are the effects that are currently available:
 	#endif
 	    RGB_MATRIX_EFFECT_MAX
 	};
+    
+You can disable a single effect by defining `DISABLE_[EFFECT_NAME]` in your `config.h`:
+
+```
+// to disable alphas mods
+#define DISABLE_RGB_MATRIX_ALPHAS_MODS
```

Comment:
One last thing.... if you could list all of the modes?  

Not necessarily in this block, but add a table that lists ALL of them, for reference?

Reply:
Sure @drashna , added a table that lists all the defines